app/models/task.py

```diff
@@ -3,4 +3,30 @@
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    goal_id = db.Column(db.Integer, db.ForeignKey(
+        'goal.goal_id'), nullable=True)
+    goal = db.relationship('Goal', back_populates='tasks')
```

Comment:
Similar to `Goal`, tasks with empty titles and descriptions may not be useful to store. Consider adding `nullable=False` to ensure every task's title and description has a value.